tests/test_library_conf.py

```diff
@@ -512,6 +512,62 @@ def test_multiple_tracecontexts(self):
         assert "_dd.p.tid=1111111111111111" not in data["request_headers"]["x-datadog-tags"]
         assert "key1=value1" in data["request_headers"]["baggage"]
 
+    def setup_multiple_tracecontexts_with_overrides(self):
+        self.r = weblog.get(
+            "/make_distant_call",
+            params={"url": "http://weblog:7777/"},
+            headers={
+                "x-datadog-trace-id": "1",
+                "x-datadog-parent-id": "1",
+                "x-datadog-sampling-priority": "2",
+                "x-datadog-tags": "_dd.p.tid=1111111111111111,_dd.p.dm=-4",
+                "traceparent": "00-11111111111111110000000000000001-1234567890123456-01",
+                "baggage": "key1=value1",
+            },
+        )
+
+    @missing_feature(
+        library="cpp",
+        reason="baggage is not implemented, also remove DD_TRACE_PROPAGATION_STYLE_EXTRACT workaround in containers.py",
+    )
+    def test_multiple_tracecontexts_with_overrides(self):
+        interfaces.library.assert_trace_exists(self.r)
+        spans = interfaces.agent.get_spans_list(self.r)
+        assert len(spans) == 1, "Agent received the incorrect amount of spans"
+
+        # Test the extracted span context
+        span = spans[0]
+        assert (
+            span.get("traceID") != "1"  # Lower 64-bits of traceparent
+        )
+        assert (
+            span.get("traceID") != "8687463697196027922"  # Lower 64-bits of traceparent
```

Comment:
I think this line can be deleted since this trace-id doesn't show up at all in this test case